.github/dependabot.yml

```diff
@@ -10,6 +10,132 @@ updates:
     schedule:
       interval: weekly
       day: sunday
+  - package-ecosystem: docker
```

Comment:
Can you split these changes into their own PR? I'm guessing they are coming from updating the build tools and are good changes, just they are unrelated to the changes included in this PR.

Reply:
Done. New PR https://github.com/open-telemetry/opentelemetry-go-contrib/pull/3522 for Dockerfile dependabot config.